cmake/macros/macro_deal_ii_package_handle.cmake

```diff
@@ -153,6 +153,17 @@ MACRO(DEAL_II_PACKAGE_HANDLE _feature _var)
       ENDIF()
     ENDFOREACH()
 
+    #
+    # Remove certain system libraries from the link interface
+    #
```

Comment:
Can you add why we do this? I can see what the code block does, but I have no idea *why* :-)

Reply:
Sure :-)